pwiz_tools/Skyline/TestPerf/SmallMolLibrariesTutorialTest.cs

```diff
@@ -0,0 +1,431 @@
+            //   �	Click the Zoom to Selection button again to see the entire 3D MS/MS spectrum.
+            RunUI(() => SkylineWindow.GraphFullScan.ZoomToSelection(false));
+            PauseForScreenShot("3D plot MSMS full range", 19);
+
+
+            //    Reimporting Data with Drift Time Filtering
+
+            // Prior to changing the settings and reimporting the data, you may want to save the current Skyline document and create a second file in order to compare the data before and after IMS filtering. To do so:
+            //    �	On the File menu, click Save As...
+            //    �	Save the file with a different name than your original Skyline document, such as �Drosophila_Lipids_Neg_IMS_Filtered�, in the tutorial folder you created.
+            RunUI(()=>SkylineWindow.SaveDocument(TestFilesDirs[0].GetTestPath(@"Drosophila_Lipids_Neg_IMS_Filtered.sky")));
+
+            //   �	From the Settings menu, click Transition Settings.
+            //   �	Click the Ion Mobility tab.
+            transitionSettingsUI = ShowDialog<TransitionSettingsUI>(() =>
+                SkylineWindow.ShowTransitionSettingsUI(TransitionSettingsUI.TABS.IonMobility));
+
+            //   �	Check Use spectral library ion mobility values when present.
+            //   �	Set the Window Type field to Resolving power.
+            //   �	In the Resolving power field, enter �50�.
+            RunUI(() =>
+            {
+                // ReSharper disable once ConditionIsAlwaysTrueOrFalse
+                transitionSettingsUI.IonMobilityControl.IsUseSpectralLibraryIonMobilities = true;
+                transitionSettingsUI.IonMobilityControl.IonMobilityFilterResolvingPower = 50;
+                transitionSettingsUI.IonMobilityControl.WindowWidthType =
+                    IonMobilityWindowWidthCalculator.IonMobilityWindowWidthType.resolving_power;
+            });
+            //   �	The Transition Settings form should now look like this:
+            PauseForScreenShot("Transition Settings: IonMobility", 20);            //The Transition Settings should now look like:
+
+
+            //   �	Click the OK button.
+            OkDialog(transitionSettingsUI, transitionSettingsUI.OkDialog);
+            doc = WaitForDocumentChange(doc);
+
+            //   The results must now be reimported with the newly applied IMS settings.
+            //   �	From the Edit menu, click Manage Results.
+            //   �	Click the Re-import button. �*� Should appear to the left of F_A_018.
+            //   �	Select M_A_001 in the Manage Results view.
+            //   �	Click the Re-import button. �*� Should appear to the left of M_A_001.
+            //   �	Click the OK button.
+            doc = SkylineWindow.Document;
```

Comment:
There is a disposable object you can put in a using() clause for short blocks that cause a change like this.